README.md

```diff
@@ -209,4 +222,4 @@ If you believe you have found a security vulnerability in this repository, pleas
 
 We have seen issues when using npm 7+ to install local symbolic linked npm packages. For this reason, we suggest using npm 6.13 for BabylonReactNative development. To install npm 6.13, run the following command:
 
-`npm install -g npm@6.13`
```

Comment:
I think Github might've deleted the newline at the end of the file? I edited this using the site UX

Reply:
sounds good